content/en/docs/policy-reports/_index.md

```diff
@@ -98,6 +98,66 @@ Entries in a policy report contain a `result` field which can be either `pass`,
 | warn   | The annotation `policies.kyverno.io/scored` has been set to `"false"` in the policy converting otherwise `fail` results to `warn`. |
 | error  | Variable substitution failed outside of preconditions and elsewhere in the rule (ex., in the pattern).                             |
 
+### Scenarios for skipped evaluations
+
+A `skip` result signifies that Kyverno decided not to fully evaluate the resource against a specific rule. This is different from a pass where the resource was evaluated and deemed compliant. A `skip` means the rule was essentially bypassed.
+
+Here's a breakdown of common scenarios resulting in a `skip`:
+
+1. **Preconditions Not Met:**
+
+This is the most frequent reason for a skip. If a rule has preconditions defined and any of the conditions within the any or all blocks evaluate to FALSE, the entire rule is skipped. Kyverno won't even attempt to apply the pattern, effectively bypassing the rule.
+
+2. **Policy Exceptions:**
+
+Kyverno allows you to define exceptions to policies using PolicyException resources. If an exception exists that matches a specific resource and rule, Kyverno will skip the rule for that resource.
+
+3. **Conditional Anchors `()` with Unmet Conditions:**
+
+When using a conditional anchor, if the condition within the anchor evaluates to FALSE, the entire rule is skipped.
```

Comment:
```suggestion
When using a conditional anchor, the corresponding section is skipped if the condition within the anchor evaluates to FALSE.
```

Reply:
The conditional block will be skipped but the peer will be evaluated if any, does it make sense to change to the above?